api/grpcserver/v2alpha1/activation_test.go

```diff
@@ -219,7 +221,11 @@ func TestActivationService_ActivationsCount(t *testing.T) {
 		require.NoError(t, atxs.Add(db, atx))
 		epoch3ATXs[i] = *atx
 	}
-	genEpoch5 := fixture.NewAtxsGenerator().WithEpochs(5, 1)
+
+	ts := time.Now().UnixNano()
+	r := rand.New(rand.NewSource(uint64(ts))).Int63n(1000)
```

Comment:
Why would this fix the issue? Could this not lead again to collisions?

We could work with fixed seeds for the two generators or set one to `time.Now().UnixNano()` and the other to `time.Now().UnixNano()+1`. The main reason we have collisions in the first place is because `time.Now()` as a terrible resolution on windows and multiple calls to it within the same 10-15 ms return the same result.

Reply:
Didn't know about such behaviour on windows, fixing :)

Reply:
❤️  I also only know because I ran into a similar issue before. Here is the MS documentation for it:

https://learn.microsoft.com/en-us/windows/win32/sysinfo/acquiring-high-resolution-time-stamps#resolution-precision-accuracy-and-stability:~:text=This%20table%20provides%20info%20on%20the%20approximate%20resolution%2C%20access%20time%2C%20and%20precision%20of%20a%20variety%20of%20clocks.%20Note%20that%20some%20of%20the%20values%20will%20vary%20with%20different%20processors%2C%20hardware%20platforms%2C%20and%20processor%20speeds.

Go uses the `PC RTC` on Windows afaik. That's why multiple calls to `time.Now()` return the same value there - mac and linux afaik don't have that problem.